apiBase/APIRequestUtils.es6.js

```diff
@@ -48,7 +48,13 @@ const requestHeaders = (apiOptions) => {
 };
 
 const requestPath = (apiOptions, path) => {
-  return `${apiOptions.origin}/${path}`;
+  let slash = '/';
```

Comment:
`const slash = path.indexOf('/') ? '/' : ''` if you want to be super confusing :trollface: